index.src.html

```diff
@@ -77,7 +77,7 @@
       text: environment settings object
       text: responsible browsing context
       text: global object; for: environment settings object; url: concept-settings-object-global
-      text: creation URL
+      text: creation URL; url: concept-environment-creation-url
```

Comment:
If you use `<a for=environment>creation URL</a>` at the point of usage this should not be needed at all.